<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>732</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>704</width>
    <height>732</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>732</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>704</width>
     <height>704</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>704</width>
      <height>704</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>-8</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Menu">
     <widget class="QLabel" name="BackgMenu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>704</width>
        <height>704</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>181</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="SimulateButton">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#SimulateButton {
background-color: transparent;
background-image: url(:/images/SimulateBtn.png);
border-image: none;
}

QPushButton#SimulateButton:pressed {
    background-image: url(:/images/SimulateClickedBtn.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AboutButton">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#AboutButton {
background-color: transparent;
background-image: url(:/images/AboutBtn.png);
border-image: none;
}

QPushButton#AboutButton:pressed {
    background-image: url(:/images/AboutClickedBtn.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfigsButton">
         <property name="minimumSize">
          <size>
           <width>179</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#ConfigsButton {
background-color: transparent;
background-image: url(:/images/ConfigsBtn.png);
border-image: none;
}

QPushButton#ConfigsButton:pressed {
    background-image: url(:/images/ConfigsClickedBtn.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QuitButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#QuitButton {
background-color: transparent;
background-image: url(:/images/QuitBtn.png);
border-image: none;
}

QPushButton#QuitButton:pressed {
    background-image: url(:/images/QuitClickedBtn.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="About">
     <widget class="QPushButton" name="MenuButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#MenuButton {
background-color: transparent;
background-image: url(:/images/Arrow.png);
border-image: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="backgAbout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>704</width>
        <height>704</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="gitCodeBtn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>625</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#gitCodeBtn {
background-color: transparent;
background-image: url(:/images/git.png);
border-image: none;
}
QPushButton#gitCodeBtn:pressed {
    background-image: url(:/images/gitClicked.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>340</y>
        <width>269</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="gitProfileBtn">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#gitProfileBtn {
background-color: transparent;
background-image: url(:/images/git.png);
border-image: none;
}
QPushButton#gitProfileBtn:pressed {
    background-image: url(:/images/gitClicked.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lkdinBtn">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#lkdinBtn {
background-color: transparent;
background-image: url(:/images/linkedin.png);
border-image: none;
}
QPushButton#lkdinBtn:pressed {
    background-image: url(:/images/linkedinClicked.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="soBtn">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#soBtn {
background-color: transparent;
background-image: url(:/images/stack.png);
border-image: none;
}
QPushButton#soBtn:pressed {
    background-image: url(:/images/stackClicked.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Configuration">
     <widget class="QPushButton" name="MenuButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#MenuButton_2 {
background-color: transparent;
background-image: url(:/images/Arrow.png);
border-image: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="backgConfig">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>14</width>
        <height>1</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>704</width>
        <height>704</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>471</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="config_1">
           <property name="font">
            <font>
             <family>Audiowide</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel#config_1{
 color: white;
}</string>
           </property>
           <property name="text">
            <string>Show paths:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showPathsBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="config_2">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Audiowide</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel#config_2{
 color: white;
}</string>
           </property>
           <property name="text">
            <string>Ants/Colony Unit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="antsNumber">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Audiowide</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="antsSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
